client/src/components/StreamField/blocks/BaseSequenceBlock.js

```diff
@@ -580,6 +588,22 @@ export class BaseSequenceBlock {
     this.blockCountChanged();
   }
 
+  initDragNDrop() {
+    this.sortable = Sortable.create(this.sequenceContainer.get(0), {
```

Comment:
I think we can adjust the `swapThreshold` to be more user friendly. The default works but it can be improved, yet it seems hard to get the right balance, so I think we can revisit this later during feature freeze.